css-transitions-2/Overview.bs

```diff
@@ -169,23 +169,38 @@ transition's <dfn>transition generation</dfn>.
 
 ## Defining [=before-change style=]: the ''@starting-style'' rule
 
-Transitions can only start during a [=style change event=] for elements which
-has a defined [=before-change style=] established by the previous
-[=style change event=], which means that newly inserted elements or elements
-that change from not [=being rendered=] to being rendered do not start
-transitions. Still, doing transitions starting from the initial rendering is a
-valid use case.
+In Level 1 of this specification, transitions can only start during a
+[=style change event=] for elements which have a defined [=before-change style=]
+established by the previous [=style change event=]. That means a transition
+could not be started on an element that was not being rendered for the previous
+[=style change event=] (see: [[css-transitions-1#starting]]).
+
+In some cases it makes sense to start transitions on newly inserted elements or
+elements that change from not [=being rendered=] to being rendered. To allow
+for that, this specification introduces ''@starting-style''.
 
 The <dfn at-rule id="at-ruledef-starting-style">@starting-style</dfn> rule is a
-conditional group rule with the implicit condition that it matches if the
-previous [=style change event=] did not establish a [=before-change style=] for
-the element whose styles are being computed. This is used to establish a
+grouping rule where the rules inside it apply if the previous
+[=style change event=] did not establish a [=before-change style=] for the
+element whose styles are being computed. This is used to establish a
 [=before-change style=] for the current [=style change event=].
 
-If any rules in ''@starting-style'' are applied, the resulting style must be
-used as the [=before-change style=] for the current [=style change event=] and
-the computed styles must be re-computed without ''@starting-style'' rules
-applied.
+Note that this means that ''@starting-style'' rules only applies to some
+elements during a computed style update, namely elements that were not rendered
+or part of the DOM during the previous [=style change event=].
+
+If any style rules in ''@starting-style'' are applied, the resulting style must
+be used as the [=before-change style=] for the current [=style change event=].
```

Comment:
I think it's worth saying explicitly that the presence of matching rules in a `@starting-style` rule causes transitions to occur on elements that otherwise couldn't have transitions because they lack a before-change style.

Reply:
Done